src/imio/updates/scripts/auth.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import print_function
+import argparse
+import logging
+import transaction
+
+from plone import api
+from Products.PluggableAuthService.interfaces.plugins import IAuthenticationPlugin
+
+LOGGER_LEVEL = 20
+
+
+def verbose(*messages):
```

Comment:
Il faudrait déplacer verbose et setup_logger dans un seul module importé dans chacun des scripts... pour be pas répéter le même code...